layouts/shortcodes/workflow-table.html

```diff
@@ -0,0 +1,10 @@
+<div class="container workflow align-items-center">
+  <div class="row d-none d-sm-flex text-center">
+    <div class="col-3"><img src="{{ .Get "image-1" }}" /><h5><strong>Condition</strong></h5></div>
```

Comment:
Given how generic these images are and that they are not critical to the flow or understanding, I think it is OK to omit them for now, but will add generic `alt` attributes since that may be helpful.

Reply:
:+1: